ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/ai2r/AI2RNodeExecutor.java

```diff
@@ -96,29 +138,41 @@ public void update()
       {
          statusMessage.getRobotMidFeetUnderPelvisPoseInWorld().set(syncedRobot.getFramePoseReadOnly(HumanoidReferenceFrames::getMidFeetUnderPelvisFrame));
 
+         // ------------ Scene --------------
```

Comment:
I'd suggest making these comments normal or using methods instead to organizer, as we do elsewhere in the code